main.go

```diff
@@ -35,13 +35,13 @@ func main() {
 }
 
 func buildTable(rootPkgName string, base, head *CoverProfile) string {
-	const tableRowSprintf = "%-80s %8s %8s %8s\n"
+	const tableRowSprintf = "%-80s  %7s  %7s  %7s\n"
 	rootPkgName += "/"
 
 	// write report header
 	var buf strings.Builder
 	buf.WriteString(fmt.Sprintf(tableRowSprintf, "package", "before", "after", "delta"))
-	buf.WriteString(fmt.Sprintf(tableRowSprintf, "-------", "------", "-----", "-----"))
+	buf.WriteString(fmt.Sprintf(tableRowSprintf, "-------", "-------", "-------", "-------"))
```

Comment:
Not sure I like the new design changes, the old one looks better IMO :frowning_face:

Reply:
it was just to have the header lines carry over `100.0%` values (which it currently does not 😄).

Reply:
Ah, I musn'tve see any of those :trollface:

Reply:
> Ah, I musn'tve see any of those

ROFL - zing!